docs/src/main/paradox/docs/releases/v1.5-release-notes.md

```diff
@@ -76,6 +76,8 @@ We have added support for multiple Content Negotiation types when querying the S
 
 - ElasticSearchView/SparqlView/CompositeView have a new field `permission` that allows clients to customize the permission needed in order to perform queries.
 - ElasticSearchView has a new field `settings`, allowing clients to specify the Elasticsearch @link:[index settings](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html#create-index-settings){ open=new }.
+- ElasticSearchViews no longer expand the value of `@type` field. The value used will be the one provided by the client.
+- ElasticSearchViews no longer inject and expand the value of `@id` field. If `@id` is missing in the payload provided by the client, it will not be present in the Elasticsearch document.
```

Comment:
I would also recommend adding: "when `sourceAsText` configuration is set to `false`"
and: "The `@id` value can be read from the Elasticsearch document id (`_id` field) which matches the resource id."